x-pack/plugins/searchprofiler/public/application/components/profile_tree/__jest__/fixtures/normalize_indices.ts

```diff
@@ -1,4 +1,4 @@
-/*eslint-disable */
+/*eslint-disable @kbn/eslint/require-license-header,spaced-comment*/
```

Comment:
I think we can add the license header to this file to remove the need for disabling the `@kbn/eslint/require-license-header` rule.

Reply:
👍 changed that CJ!